app-misc/ledit/ledit-2.04.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
+inherit eutils
```

Comment:
eutils doesn't seem to be used in this ebuild, you can just drop this inherit line altogether.